src/osa/scripts/gain_selection.py

```diff
@@ -311,6 +309,14 @@ def apply_gain_selection(date: str, start: int, end: int, tool: str = None, no_q
                     sp.run(["cp", file, output_dir])
 
 
+def run_summary_table(date: str) -> Table:
```

Comment:
there should be a similar function defined somewhere in the code already that might be used for this:

https://github.com/cta-observatory/lstosa/blob/e148c8c61fcec0f51cbdf1399f042a4fb0b302bd/src/osa/nightsummary/nightsummary.py#L47-L79